### ISSUE TITLE
[Serious] Active user interface component lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#575]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87616 |
| **Checkpoint** | 1.4 Distinguishable › Non-Text Contrast - Active User Interface Components (1.4.11.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Settings Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/06/26 at 5:23 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The visual boundary of an active user interface component lacks 3 to 1 contrast ratio.
Global: The border around text edit boxes is very light gray, making it difficult to see where to click to interact with the field.

## Recommendation

Rule : 
The visual characteristics that indicate the component's purpose or type (e.g. is it a checkbox, text field, slider, switch, etc.) must have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.

How To Fix: 
Fix this issue by adjusting the user interface component boundary and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.

Reference: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast

Background: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.

## Screenshots

![01_user interface contrast](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87616/01_user_interface_contrast.png)
